cluster/kube/builder/workload.go

```
@@ -408,6 +409,50 @@ func (b *Workload) imagePullSecrets() []corev1.LocalObjectReference {
+func (b *Workload) collectIngressURIs() []string {
+	var ingressURIs []string
+	lid := b.deployment.LeaseID()
+
+	// Collect all ingress URIs from all services in the deployment
+	for _, service := range b.group.Services {
+		for _, expose := range service.Expose {
+			// Check if this is an ingress-capable expose (not global, but accessible via ingress)
+			if expose.IsIngress() {
+				protocol := "http"
+				if expose.Proto == "tcp" && expose.Port == 443 {
+					protocol = "https"
```

golangci-lint (1.64.8)

string `https` has 3 occurrences, make it a constant

(goconst)